source/code/plugins/oms_common.rb

```diff
@@ -922,7 +928,8 @@ def refresh_cache
         @cache_lock.synchronize {
           @condition.wait(@cache_lock, @refresh_interval_seconds)
           # Flush the cache completely to prevent it from growing indefinitly
-          @cache = {}
+          # Disabling refresh functionality to prevent data loss btw rsyslog and OMS
+          # @cache = {}
```

Comment:
This appears to be the only cache invalidation policy. Disabling this means the cache will contain more and more invalid data over time.